packages/esm-patient-orders-app/src/order-basket/order-basket.workspace.tsx

```diff
@@ -52,34 +54,41 @@ const OrderBasket: React.FC<DefaultPatientWorkspaceProps> = ({
     const abortController = new AbortController();
     setCreatingEncounterError('');
     let orderEncounterUuid = encounterUuid;
-    // If there's no encounter present, create an encounter for the order.
+    setIsSavingOrders(true);
+    // If there's no encounter present, create an encounter along with the orders.
     if (!orderEncounterUuid) {
       try {
-        orderEncounterUuid = await createEmptyEncounter(
+        await postOrdersOnNewEncounter(
           patientUuid,
           config?.orderEncounterType,
           activeVisitRequired ? activeVisit : null,
           session?.sessionLocation?.uuid,
           abortController,
         );
         mutateEncounterUuid();
+        clearOrders();
+        await mutateOrders();
+        closeWorkspaceWithSavedChanges();
+        showOrderSuccessToast(t, orders);
       } catch (e) {
-        setCreatingEncounterError(e);
         console.error(e);
-        return; // don't try to create the orders if we couldn't create the encounter
+        setCreatingEncounterError(
+          e.responseBody.error.message ||
```

Comment:
> Thanks for the follow up @jayasanka-sack , however all errors are displayed by an inline notification on the UI. Not sure why it is skipping that on the tests

@usamaidrsk I don't think it has anything to do with the test. The issue lies on this line I guess. In this specific instance theres no `error` property on the `e.responseBody`.

You can find it's staten on the error log:
```
Unhandled rejection:  TypeError: Cannot read properties of undefined (reading 'error')

```
```
showToast must be called with an object having a 'description' property that is a non-empty string or object
Cannot read properties of undefined (reading 'error')
```